docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -426,6 +426,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
+------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "4/5ths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+This rule is an implementation of the demographic parity ratio introduced
+in :ref:`disparity_metrics` section, with a result considered 'fair' if the ratio
+exceeds 80% for all identified subgroups.
+*Unthinking application of this threshold is an extremely common abstraction trap.*
+
+The trap springs on three levels:
+
+1. Where is it valid?
+2. What does it mean, legally?
+3. Why 80%?
+
+*Where is the 4/5ths rule valid?*
+The answer is that it is only valid
+within a specific area of US federal employment law.
+Outside this context, the rule has no particular claim to validity.
+
+*What exactly does the rule mean?*
+Legally, the 80% threshold is a limit for
+`prima facie evidence <https://en.wikipedia.org/wiki/Prima_facie>`_
+that illegal discrimination has occurred.
```

Comment:
I would add some statements about using the statistic compared to a chosen, relevant (control) population and how a causal link also needs to be identified in the *prima facie case*